<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Watcher</class>
 <widget class="QWidget" name="Watcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watcher</string>
  </property>
  <widget class="QWidget" name="subplt1" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <widget class="QTextEdit" name="text1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="subtitle1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="subplt2" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="subtitle2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="subSlider1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="SyncCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
  </widget>
  <widget class="QSlider" name="subSlider2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>pressed()</signal>
   <receiver>Watcher</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
